wsgi/statuses.json

```diff
@@ -3,13 +3,7 @@
     "global_status": "scheduled",
     "global_verbose_status": "There are scheduled downtimes in progress",
     "services": {
-        "ask": {
-            "message": "Everything seems to be working.",
-            "name": "Ask Fedora",
-            "status": "good",
-            "url": "http://ask.fedoraproject.org/"
-        },
-        "authentication": {
+       "authentication": {
```

Comment:
there are a few lines in this diff that show a little under-indented compare to what was here before, is this expected?

Reply:
Thanks pypingou! I missed that. Will fix.